.github/workflows/release.yaml

```diff
@@ -94,10 +94,3 @@ jobs:
           asset_path: schema-loader/build/libs/scalardb-schema-loader-${{ steps.version.outputs.version }}.jar
           asset_name: scalardb-schema-loader-${{ steps.version.outputs.version }}.jar
           asset_content_type: application/java-archive
-
-      - name: Cleanup Gradle cache
```

Comment:
OK to delete them? (I think it's auto-generated but not really sure why they are auto-generated.)

Reply:
Actually, I'm also not sure they are needed, so I deleted it. I will restore it if we have any problems with it. Thank you.